utils/wasm-builder/src/wasm_project.rs

```diff
@@ -173,6 +173,9 @@ impl WasmProject {
         let to_opt_path = self
             .target_dir
             .join(format!("{}.opt.wasm", &file_base_name));
+
+        let _ = crate::optimize::optimize_wasm(to_path.clone(), "s", false);
```

Comment:
Why don't we skip debug symbols here?

Reply:
symbols are stripped when creating a meta and opt file, it is convenient to analyze a file in a regular wasm with tools like `twiggy` with debug symbols.